src/plugins/partner_sku_reviews.coffee

```diff
@@ -0,0 +1,1326 @@
+  return element.addEventListener event, handler, false  if element.addEventListener
+  element.attachEvent "on#{event}", handler if element.attachEvent
+
+format_1_decimal = (num, translate_decimal=true) ->
+  decimal_point = if translate_decimal then '@@translations.sku_reviews.decimal_point' else '.'
+  num.toFixed(1).replace('.', decimal_point)
+
+esc = (unsafe) ->
+    return unsafe
+         .replace(/&/g, "&amp;")
+         .replace(/</g, "&lt;")
+         .replace(/>/g, "&gt;")
+         .replace(/"/g, "&quot;")
+         .replace(/'/g, "&#039;");
+
+## Reusable template fragments
+
+starText = (star_score, fraction=false, translate_decimal=true) ->
+  star_score = format_1_decimal(star_score, translate_decimal) if fraction
+  star_score + if !fraction && star_score == 1
+    ' @@translations.sku_reviews.star'
+  else
+    ' @@translations.sku_reviews.stars'
+
+stars = (reviewscore, reviews_count, fraction=false) ->
+  starsWidth = 0;
+  if reviewscore
+    starsWidth = (reviewscore * 20).toFixed(1)
+    title = starText(reviewscore, fraction)
+
+  if reviews_count
+    title += " @@translations.sku_reviews.from #{reviews_count} "
+    title += if reviews_count == 1
+               ' @@translations.sku_reviews.user_from '
+             else
+               '@@translations.sku_reviews.users_from'
+
+  """
+    <div class="sa-star-rating" title="#{title}">
+      <div class="sa-star-rating-wrapper">
+        <div class="sa-actual-star-rating" style="width: #{starsWidth}%"></div>
+      </div>
+    </div>
+  """
+
+starRating = (reviewscore, reviews_count, translate_decimal=true) -> """
+    <div class="sa-rating-value">#{format_1_decimal(reviewscore, translate_decimal)}</div>
+    #{stars(reviewscore, reviews_count, true)}
+    <div class="sa-reviews-count-compact">#{reviews_count}</div>
+  """
+
+brandIndicator = -> """
+    <div class="sa-brand">
+      <div class="sa-brand-inner">
+        <span class="sa-brand-powered-by">Powered by</span>
+        <div class="sa-brand-logo"></div>
+      </div>
+    </div>
+  """
+
+ratingBreakdown = (breakdown) ->
+  list_elements = breakdown.map ({star, percentage, count}) ->
+    """
+      <li class="sa-rating-breakdown-class">
+        <div class="sa-num-stars">
+          #{starText(star)}
+        </div>
+        <div class="sa-rating-bar-wrap">
+          <div class="sa-rating-bar" style="width: #{percentage}%"></div>
+        </div>
+        <div class="sa-rating-count">#{count}</div>
+      </li>
+    """
+
+  """
+    <ul class="sa-rating-breakdown">
+      #{list_elements.join('')}
+    </ul>
+  """
+
+skuSales = (sales_text) ->
+  return '' unless sales_text
+
+  """
+    <div class="sa-sales-wrap">
+      <img class="sa-sales-icon" alt="sales icon"/>
+      <span class="sa-sales-text">#{sales_text}</span>
+    </div>
+  """
+
+reviewsAggregation = (aggregation) ->
+  return '' unless aggregation && aggregation.length
+
+  aggregation_elements = aggregation.map ({label, score, style}) ->
```

Comment:
[Minor] Let's add a comment for the supported styles, `sa-good`, `sa-so-so`, `sa-bad`

Reply:
Comments in general..